bundles/org.openhab.binding.airq/src/main/java/org/openhab/binding/airq/internal/airqHandler.java

```diff
@@ -0,0 +1,986 @@
+
+    final class ResultPair {
+        private final float value;
+        private final float maxdev;
+
+        public float getvalue() {
+            return value;
+        }
+
+        public float getmaxdev() {
+            return maxdev;
+        }
+
+        // ResultPair() expects a string formed as this: [1234,56,789,012] and gives back a ResultPair
+        // consisting of the two numbers
+        public ResultPair(String input) {
+            value = Float.parseFloat(input.substring(1, input.indexOf(',')));
+            maxdev = Float.parseFloat(input.substring(input.indexOf(',') + 1, input.length() - 1));
+        }
+    }
+
+    public airqHandler(Thing thing) {
+        super(thing);
+        ipaddress = "";
+        password = "";
+    }
+
+    public Boolean ohCmd2airqCmd(String ohcmd) {
+        switch (ohcmd) {
+            case "ON":
+                return true;
+            case "OFF":
+                return false;
+        }
+        return false;
+    }
+
+    private boolean isTimeFormat(String str) {
+        try {
+            LocalTime.parse(str);
+        } catch (DateTimeParseException e) {
+            return false;
+        }
+        return true;
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        logger.info(
+                "air-Q - airqHandler - handleCommand(): request received to handle value {} command {} of channelUID={}",
+                command, command.getClass(), channelUID);
+        if ((command instanceof OnOffType) || (command instanceof StringType)) {
+            JsonObject newobj = new JsonObject();
+            JsonObject subjson = new JsonObject();
+            switch (channelUID.getId()) {
+                case "wifi":
+                    // we do not allow to switch off Wifi because otherwise we can't connect to the air-Q device anymore
+                    break;
+                case "wifiInfo":
+                    newobj.addProperty("WifiInfo", ohCmd2airqCmd(command.toString()));
+                    changeSettings(newobj);
+                    break;
+                case "fireAlarm":
+                    newobj.addProperty("FireAlarm", ohCmd2airqCmd(command.toString()));
+                    changeSettings(newobj);
+                    break;
+                case "cloudUpload":
+                    newobj.addProperty("cloudUpload", ohCmd2airqCmd(command.toString()));
+                    changeSettings(newobj);
+                    break;
+                case "autoDriftCompensation":
+                    newobj.addProperty("AutoDriftCompensation", ohCmd2airqCmd(command.toString()));
+                    changeSettings(newobj);
+                    break;
+                case "autoUpdate":
+                    // note that this property is binary but uses 1 and 0 instead of true and false
+                    newobj.addProperty("AutoUpdate", command.toString() == "ON" ? 1 : 0);
```

Comment:
```suggestion
                    newobj.addProperty("AutoUpdate", command == OnOffType.ON ? 1 : 0);
```
No need to convert to a string.